data/web/doc_old_layout_redirection.conf

```diff
@@ -0,0 +1,6 @@
+
+RewriteEngine On
+RewriteCond %{REQUEST_URI} !^/pulpcore
+RewriteCond %{REQUEST_URI} !^/pulp_
```

Comment:
can you also add !^/en/2 to the rules?

Reply:
That's for Pulp 2 docs that are not part of pulpcore.

Reply:
@duck-rh ^ ... The redirects are working great. That's the only additional change we need before merging this.

Reply:
@dkliban added